config/initializers/rollbar.rb

```diff
@@ -0,0 +1,57 @@
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+
+  # Here we'll disable in 'test':
+  if Rails.env.test? || Rails.env.development?
```

Comment:
Do we want this, or do we want it configurable via env?

Reply:
This is fine. In dev and test you'd just look at exceptions as they come up in your console.